src/tracing/node_trace_buffer.h

```diff
@@ -75,7 +75,7 @@ class NodeTraceBuffer : public TraceBuffer {
   // Used to wait until async handles have been closed.
   ConditionVariable exit_cond_;
   std::unique_ptr<NodeTraceWriter> trace_writer_;
-  // TODO: Change std::atomic to something less contentious.
+  // TODO(misterpoe) : Change std::atomic to something less contentious.
```

Comment:
As I've written in https://github.com/nodejs/node/issues/14490#issuecomment-318602020, I'd put `matthewloring` instead. Also, remove the space between `)` and `:`.